### PR TITLE
Feature add timber integration

### PR DESCRIPTION
**Pull request by idosun**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,12 +1,18 @@
 SENTRY_ORG=testorg-az
 SENTRY_PROJECT=paranoid-android
-RELEASE=`sentry-cli releases propose-version`
+#RELEASE=`sentry-cli releases propose-version`
+RELEASE=40.0.0
 
-all: gradle_build setup_release
+all: gradle_build setup_release upload_debug_files
```
Comment (thinkocapo) [inline Makefile:6]: wasn't this already being taken care of by something else, if I'm not mistaken?

or did you find crashes weren't symbolicated, or you were lacking stack traces?

File: app/build.gradle [hunk 2]
```diff
@@ -14,8 +14,8 @@ android {
         applicationId "com.example.vu.android"
         minSdkVersion 21
         targetSdkVersion 29
-        versionCode 17
-        versionName "1.7"
+        versionCode 15
```
Comment (thinkocapo) [inline app/build.gradle:17]: ok this is not the SDK's version. Is this the Android API version or our App's version? Was this downgrade needed for TImber to work?

File: app/build.gradle [hunk 3]
```diff
@@ -44,18 +44,39 @@ android {
...
 sentry {
+// Disables or enables the automatic configuration of ProGuard
+    // for Sentry.  This injects a default config for ProGuard so
+    // you don't need to do it manually.
+    autoProguardConfig true
+
+    // Enables or disables the automatic upload of mapping files
+    // during a build.  If you disable this, you'll need to manually
+    // upload the mapping files with sentry-cli when you do a release.
+    autoUpload true
```
Comment (thinkocapo) [inline app/build.gradle:69]: if it's `autoUpload true` then does that mean we don't need to use sentry-cli for the `upload_debug_files` defined in the Makefile?

**Review by thinkocapo — CHANGES_REQUESTED** [review]
Was capturing native crashes by NDK overall broken, you found? Is that why there’s a lot of updates for NDK?

Can you put a few links to your Sentry Issues (from Timber, and anything with the new CustomerType tag, and definitely the Native Crash from NDK).

File: app/build.gradle (hunk 3 above)
Comment (idosun) [reply app/build.gradle:69]: autoUpload -  is for proguard mapping files that are then uploaded to project settings > proguard mapping

**Comment by idosun** [comment]
will create new PR

**Pull request closed without merging**